### PR TITLE
Update metadata/Architecutre: Mobirise.Mobirise version 5.3.10

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21638)

### CODE REVIEW

File: manifests/m/Mobirise/Mobirise/5.3.10/Mobirise.Mobirise.installer.yaml [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
 Installers:
-- Architecture: neutral
+- InstallerLocale: en-US
+  Architecture: x86
   InstallerType: nullsoft
+  Scope: machine
```
Comment (OfficialEsco) [inline manifests/m/Mobirise/Mobirise/5.3.10/Mobirise.Mobirise.installer.yaml:14]:
```suggestion
  Scope: user
```
quick fix 😬

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/m/Mobirise/Mobirise/5.3.10/Mobirise.Mobirise.installer.yaml (hunk 1 above)
Comment (quhxl) [reply manifests/m/Mobirise/Mobirise/5.3.10/Mobirise.Mobirise.installer.yaml:14]: folder

This is the install location...
@OfficialEsco

**Review by OfficialEsco — APPROVED** [review]
Seems like you are correct...
image

It does however not require Administrator to install in some weird way

**Comment by quhxl** [comment]
> It does however not require Administrator to install in some weird way

This has happened to me several times. Weird installers with weird permissions....

**Pull request merged**